### ISSUE TITLE
Version 0.3.1 Breaks Google Closure Compiler Support

### ISSUE THREAD

**Issue by xcthulhu** [bitpay/bitauth#55]
I'm sorry to report that the latest version has broken the support for the Google Closure Compiler that previous versions provided.

This is a blocker for https://github.com/cljsjs/packages/pull/255 , which is in turn a blocker for getting unicode support into my Clojure(Script) port https://github.com/xcthulhu/clj-bitauth .

I am using the following as my [extern](https://developers.google.com/closure/compiler/docs/api-tutorial3?hl=en) file:

```js
var BitAuth = {
    "getPublicKeyFromPrivateKey": function (privateKey) {},
    "getSinFromPublicKey": function (publicKey) {},
    "generateSin": function () {},
    "sign": function (data, privateKey) {},
    "verifySignature": function (data, publicKey, hexSignature, callBack) {},
    "validateSin": function (sin, callBack) {}
};
```

Perhaps BitPay should host the extern file, since you own the API?  At any rate I'm bummed that this doesn't work and I have to resort to the old version without UTF-8 support 😕

**Comment by xcthulhu** [comment]
*bump*

Just wanted to point out that this is fixed in #56

**Comment by xcthulhu** [comment]
This extern file doesn't work, I am closing this.
